Controller/StartPayment/Index.php

```diff
@@ -66,7 +66,7 @@ public function execute() {
 		$billingAddress = $order->getBillingAddress();
 
 		$response = $this->gateway->createPayment(array_merge([
-			'amount' => intval($order->getGrandTotal()) * 100,
+			'amount' => intval(round($order->getGrandTotal() * 100)),
```

Comment:
what about using int cast here? we did the same in [prestashop](https://github.com/paypronl/prestashop-payments-plugin/pull/2/files), the reason is I think it works faster, here is also an [article](https://tonyshowoff.com/articles/casting-int-faster-than-intval-in-php/)

Reply:
nice, thank you!